OPHD/UI/Core/Slider.cpp

```diff
@@ -1,5 +1,5 @@
 /**
- * \file	Slider.c++
+ * \file	Slider.cpp
  * \brief	Functions for sliding controls.
  * \author	Goof
```

Comment:
Bit of a side note, though I've read that when author's are listed in the source file of open source projects, it tends to discourage other people form maintaining the file. Perhaps we should omit author comments from source files. The information is still part of the commit meta data. Similar for comments that simply state the date a change was made.

Reply:
... of course, I've never felt comfortable removing someone else's author comments. But then, that's maybe getting to the heart of the above.

Reply:
I had mixed thoughts about this as well. I think you are right about removing attribution unless it is a large module like the Pathfinder or something.

Reply:
Author notes are an artifact of the past when source control wasn't a thing. Author information is in the repository commit history, I see very low value in keeping authorship comments in place. Including the file name also makes no real sense.